specs/rm/requirements-management-spec.html

```diff
@@ -871,11 +871,23 @@ <h2>Change History</h2>
           </td>
         </tr>
         <tr>
-          <td>01</td>
-          <td>2018-05-22</td>
-          <td>Jad El-khoury</td>
+          <td>csprd01</td>
+          <td>2018-05-31</td>
+          <td>Mark Schulte and Jad El-khoury</td>
           <td>Initial Committee Specification Draft migrated from open-services.net</td>
         </tr>
+        <tr>
+          <td>cs01</td>
+          <td>2018-08-24</td>
+          <td>Mark Schulte and Jad El-khoury</td>
+          <td>Committee Specification 01</td>
+        </tr>
+        <tr>
+          <td>psd02</td>
+          <td>2019-10-01</td>
+          <td>Mark Schulte and Jad El-khoury</td>
+          <td>Project Specification Draft 02</td>
```

Comment:
Can you describe any upcoming PS01 changes? Also, I think it's only you after the OP transition as Mark did not join the OP

Reply:
better now?